src/verify.c

```diff
@@ -621,25 +621,18 @@ static bool parse_opt(int opt, char *optarg)
 	case 'w':
 		cmdline_option_picky_whitelist = strdup_or_die(optarg);
 		return true;
-	case 'B': {
-		/* This option should only be available to verify for backward compatibility */
-		if (!cmdline_command_verify) {
-			print("diagnose: invalid option -- 'B'\n");
-			return false;
-		}
-		char *arg_copy = strdup_or_die(optarg);
-		char *token = strtok(arg_copy, ",");
-		while (token) {
-			cmdline_option_bundles = list_prepend_data(cmdline_option_bundles, strdup_or_die(token));
-			token = strtok(NULL, ",");
-		}
-		free(arg_copy);
+	case 'B':
+		/* if we are parsing a list from the command line we don't want to append it to
+		 * a possible existing list parsed from a config file, we want to replace it, so
+		 * we need to delete the existing list first */
+		list_free_list(cmdline_option_bundles);
+		cmdline_option_bundles = NULL;
```

Comment:
You're setting the same variable twice. you don't need this line

Reply:
haha, dah! ☺️